### DISCUSSION TITLE
Time-constraints during calibration

### DISCUSSION THREAD

**Discussion by isgiddy** [OceanGlidersCommunity/Oxygen_SOP#193] [Q&A] [answered]
For each temperature run should we leave all materials, reagents and sensors in the lab/workshop/room overnight (or as recommended for 8 hours).

When time is limited, is it ok to begin calibration when the sensor equilibrates (if this occurs in a time period shorter than 8 hours)?

@patricialg @tomhull @hahn-johannes

**Comment by tomhull** [comment, marked as answer]
There are two things here.
The general idea is that if your instrument and gear is warming/cooling rapidly it's difficult to get a good calibration. I've found though that unless you're in a temperature controlled room (not a bad idea) you're going to get temperature fluctuations during the day, I would say that if the sensor (both oxygen and temperature) is stable over 10 minutes it would be fine.

The other part, and perhaps more important, is that the foils need to be fully hydrated, which does appear to take some time, at least 1 hour if it was dry to begin with. It is _probably_ fine to do the calibration after one hour if the sensor was kept wet, but I have not actually tested it and generally I put the sensors in a bucket overnight prior to calibrating.

**Comment by soerenthomsen** [comment]
Agree with what Tom is writing. Hope the pre deployment calibration went well.
